src/app/tabs/compile-tab.js

```diff
@@ -303,7 +300,7 @@ function compileTab (container, appAPI, appEvents, opts) {
     }
     return el
   }
-  return el
+  return { render () { return el } }
```

Comment:
same here could we put `render` as a function of compile tab?

Reply:
of course - i will make pull requests to refactor all the tabs to have them as proper prototype functions of the class that the `compile-tab` will then be.

This is just a quick hack to have minimum change to standardize the API - it's not meant to stay like that.

I did it in order to make small pull requests.

---

If I change it here, then maybe i should change it for all the tabs, which will probably add 3-5 or more lines of code to each of them ... :/

**so do you think it would be ok to leave it like that - because it's working - and then fix that with the next upcoming PRs?**

Reply:
ah I said above that this is fine, so we can merge this as long as you'll make another PR to make proper `render` function

Reply:
ok